internal/action/bufpane.go

```diff
@@ -738,6 +738,7 @@ var BufKeyActions = map[string]BufKeyAction{
 	"CursorRight":               (*BufPane).CursorRight,
 	"CursorStart":               (*BufPane).CursorStart,
 	"CursorEnd":                 (*BufPane).CursorEnd,
+	"CursorToView":              (*BufPane).CursorToView,
```

Comment:
It should be also mentioned in `runtime/help/keybindings.md`.

Reply:
Good point.